schema.sql

```diff
@@ -11,4 +11,24 @@ CREATE TABLE animals (
 
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly add a new table with the name `vets`.
- [ ] As stated in the project requirements, kindly add a new join table with the name `specializations`.
- [ ] As stated in the project requirements, kindly add a new join table with the name `visits`